lib/galaxy/config.py

```diff
@@ -305,7 +305,7 @@ def __init__( self, **kwargs ):
             self.use_tool_dependencies = True
         else:
             self.tool_dependency_dir = None
-            self.use_tool_dependencies = False
+            self.use_tool_dependencies = os.path.exists(self.dependency_resolvers_config_file)
```

Comment:
I think that the default configuration may stop working after this change if the admin does not copy `config/dependency_resolvers_conf.xml.sample` to `config/dependency_resolvers_conf.xml` .